bin/codemods/src/single-tree-rendering.js

```diff
@@ -35,14 +43,30 @@ export default function transformer( file, api ) {
 	}
 
 	/**
+	 * Check if `parameters` has `param` either as a string or as a name of
+	 * an object, which could be e.g. an `Identifier`.
+	 *
+	 * @param {array} parameters Parameters to look from
+	 * @param {string} parameter Parameter name
+	 * @returns {boolean}
+	 */
+	function hasParam( parameters, parameter ) {
+		return (
+			_.findIndex( parameters || [], parameter => {
```

Comment:
I'd make the `[]` the default value for `hasParam`'s `parameters` arg.

Reply:
Right on, I'm clearly working too much with es5 files at work... 🙃